week-8/Homework/mandatory/1-practice/1-practice.md

```diff
@@ -20,12 +20,19 @@ Enjoy!
 
 The following endpoint is publicly available from Github
 
-    GET https://api.github.com/repos/{owner}/{repo}/pulls/{pull_number}/comments
+    GET https://api.github.com/repos/CodeYourFuture/js-exercise/pulls/688/comments
 
 1. Describe in a sentence what this API endpoint returns.
 
 <!-- Write your answer here -->
+<!--This API return an array of objects from pull_number of repo of owner-->
```

Comment:
Note that when you put these in comments, they're not visible in the rendered version: https://github.com/naderakhgari/js-exercises/blob/nader-js-3-week-2/week-8/Homework/mandatory/1-practice/1-practice.md

Reply:
thanks for your notice. I got it.